ts/Stock/StockToolsGui.ts

```diff
@@ -1411,12 +1411,15 @@ class Toolbar {
             toolbar,
             wrapper;
 
+        // #15593
+        css(container.parentNode, { position: 'relative' });
```

Comment:
Usually, when we set hardcoded styles, there is a case where users come back and say they need to change it. I think it would make sense to add `stockTools.gui.style` option with this predefined  `{ position }` config

Reply:
Calling it just `style` makes it sound like it gets applied to the stocktools and not `renderTo`, should it perhaps be called something like `renderToStyle`?

Reply:
Oh, I thought you added this style to the toolbar container, not the chart's container. I'm not sure if can we do this in [`chart.style`](https://api.highcharts.com/highcharts/chart.style)? So the module would just set this config. I don't know how would we inform users that the Stock Tools module adds this style. Perhaps @TorsteinHonsi has some ideas here

Reply:
It's dangerous to fiddle too much with the `renderTo`, as it's the implementer's element. Is there a reason why we are not using the `container` itself? It already has `position: relative`.